### PR TITLE
Get acquire-common building

### PR DESCRIPTION
**Pull request by aliddell**
Includes all commits from acquire-core-libs, acquire-video-runtime, and acquire-driver-common.

- Removed the PR template. It was a mistake.
- Added `@file` docstring directive and brief description to all tests.
- Moved `write-side-by-side-tiff.cpp` to tests under acquire-driver-common (previously under acquire-video-runtime).
- Did a little additional validation in `change-external-metadata.cpp`.
- The base CMakeLists.txt files for each of the subprojects were all simplified, since there's a new base project where all that can be handled.
- CPack builds one big distributable, but it includes all the static libraries, headers, and the acquire-driver-common DLL.
  - Note that git versioning no longer works the way it used to, because we don't have separate git histories or tags for subdirectories, i.e., acquire-core-libs, acquire-video-runtime, and acquire-driver-common will all have the same versions from now on. I propose we go to 0.2.0 to reflect this change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+#### Camera devices
+
+- **simulated: uniform random** - Produces uniform random noise for each pixel.
+- **simulated: radial sin** - Produces an animated radial sin-wave pattern.
+- **simulated: empty** - Produces no data, leaving a image buffers blank. Simulates going as fast as possible.
+
+#### Storage devices
+
+- **raw** - Streams to a raw binary file.
+- **tiff** - Streams to a [bigtiff][] file. Metadata is stored in the `ImageDescription` tag for each frame as a `json`
```
Comment (andy-sweet) [inline README.md]: Probably not new, but the extra `[]` is not needed.

```suggestion
- **tiff** - Streams to a [bigtiff] file. Metadata is stored in the `ImageDescription` tag for each frame as a `json`
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  string.
+- **tiff-json** - Stores the video stream in a *bigtiff* (as above) and stores metadata in a `json` file. Both are
+  located
+  in a folder identified by the `filename` property.
```
Comment (andy-sweet) [inline README.md]: Probably not new, but this is a weird line break.

```suggestion
  located in a folder identified by the `filename` property.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+## Acquire Common Driver
+
+This is an Acquire Driver that exposes commonly used devices.
+
+#### Camera devices
+
+- **simulated: uniform random** - Produces uniform random noise for each pixel.
+- **simulated: radial sin** - Produces an animated radial sin-wave pattern.
+- **simulated: empty** - Produces no data, leaving a image buffers blank. Simulates going as fast as possible.
```
Comment (andy-sweet) [inline README.md]:
```suggestion
- **simulated: empty** - Produces no data, leaving image buffers blank. Simulates going as fast as possible.
```

File: CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+include(cmake/ide.cmake)
+include(cmake/msvc.cmake)
+include(cmake/simd.cmake)
+
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_CXX_STANDARD 20)
+
+aq_require(acquire-core-libs)
```
Comment (andy-sweet) [inline CMakeLists.txt]: After this reorg, do we still need `aq_require` or can we just use `add_subdirectory` directly now?

**Review by andy-sweet — APPROVED** [review]
Generally looks great. Approving because CI is passing and I was able to build, test, and install successfully.

Most of my comments are small suggestions or questions.

File: CMakeLists.txt (hunk 4 above)
Comment (aliddell) [reply CMakeLists.txt]: We probably could get away with just `add_subdirectory`.

**Pull request merged**